packages/bootleg-tokens/contracts/ISharedRoyaltyTokenEnumerable.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.5.3;
+
+/**
+ * @title Shared Royalty NFT Enumerable interface for franchisors
+ */
+contract ISharedRoyaltyTokenEnumerable {

```

Comment:
Does it make sense to enumerate the payments too? A downstream app could want to graph payments over time? Maybe do price analysis?

<img src="https://i.kym-cdn.com/photos/images/original/000/297/088/4d6.png" width=200 />

Reply:
Sure. Can do that too. Why not.